src/action-bar.scss

```diff
@@ -72,42 +92,49 @@ $block: #{$fd-namespace}-action-bar;
     @include fd-hide-on-mobile();
 
     position: relative;
-    margin-right: fd-space(5);
-
-    &::after {
-      content: '';
-      width: 0;
-      height: 100%;
-      position: absolute;
-      top: 0;
-      right: - fd-space(1);
-      border-right-style: solid;
-      border-right-width: 1px;
-
-      @include fd-var-color("border-right-color", $fd-action-bar-backbtn-separator-color, --fd-action-bar-backbtn-separator-color);
-    }
+    margin-right: 0.5rem;
 
     @include fd-rtl() {
       transform: scaleX(-1);
-      margin-left: fd-space(5);
+      margin-left: 0.5rem;
       margin-right: 0;
     }
   }
 
   &__title {
     @include fd-reset();
     @include fd-heading();
-    @include fd-type("4");
-    @include fd-action-bar-responsive();
 
-    margin-bottom: fd-space(base);
+    font-size: var(--sapFontHeader3Size);
+    font-family: var(--sapFontHeaderFamily);
+    font-weight: normal;
```

Comment:
Shouldn't the font-weight be in the reset?

Reply:
See https://github.com/SAP/fundamental-styles/blob/master/src/mixins/_mixins.scss#L10

Reply:
Yes you're right. Fixed this by removing font-weight:normal;